mobile/about.html

```diff
@@ -12,6 +12,47 @@ <h1>About this document</h1>
     <main data-contents="about">
       <section>
         <h2>Change History</h2>
+        <section>
+          <h3>November 2019</h3>
+          <p>The <a href="https://www.w3.org/2019/11/web-roadmaps/mobile/">November 2019</a> snapshot adds a few new proposals being incubated in the Web Platform Incubator Community Group, some new deliverables from various W3C working groups, and refreshes the status of other specifications. Changes made since the April 2019 snapshot:</p>
+          <dl>
+            <dt>Exploratory work</dt>
+            <dd><ul>
+              <li>Mention WebTransport in <a data-page="network"></a></li>
+              <li>Mention isInputPending in <a data-page="performance"></a> and <a data-page="userinput"></a></li>
+              <li>Mention the virtual-scroller proposal in <a data-page="performance"></a></li>
+              <li>Mention Input for workers and worklets in <a data-page="userinput"></a></li>
+              <li>Mention Audio Device Client in <a data-page="performance"></a></li>
+              <li>Mention new performance metric interfaces in <a data-page="performance"></a></li>
+            </ul></dd>
+
+            <dt>Technologies in progress</dt>
+            <dd><ul>
+              <li>Mention Web components in <a data-page="adaptation"></a></li>
+              <li>Mention the Scalable Video Coding (SVC) Extension for WebRTC in <a data-page="media"></a></li>
+              <li>Mention two WebXR modules in <a data-page="media"></a></li>
+              <li>Move CSS Animation Worklet API to technologies in progress in <a data-page="performance"></a></li>
+              <li>Move specifications now in Media WG to technologies in progress in <a data-page="media"></a></li>
+            </ul></dd>
+
+            <dt>Well-deployed technologies</dt>
+            <dd><ul>
+              <li>Move CSS Containment Module Level 1 to well-deployed technologies in <a data-page="performance"></a></li>
+            </ul></dd>
+          </dl>
+          <p>The implementation info has also been updated to reflect known implementation status in November 2019. Some of these updates may have been triggered by updates to underlying platform status sites. Main updates are:</p>
+          <ul>
+            <li>Resize Observer shipped in Firefox and Safari, see <a data-page="adaptation"></a></li>
+            <li>Source capabilities in Media Capture and Streams shipped in Chrome, see <a data-page="adaptation"></a></li>
+            <li>datalist element in HTML shipped on all main browsers, see <a data-page="forms"></a></li>
+            <li><code>StorageManager</code> shipped in Firefox (desktop only), see <a data-page="storage"></a></li>
```

Comment:
If you wrap `StorageManager` in `<code></code>`, which seems a good idea, I think you should also do the same wrapping for other code-like references, notably `isInputPending` and `virtual-scroller` (which I would actually also list as `<virtual-scroller>`)

Reply:
Added. Thanks!